hazelcast/src/main/java/com/hazelcast/cache/impl/ICacheService.java

```diff
@@ -90,4 +91,8 @@
     String addInvalidationListener(String name, CacheEventListener listener);
 
     void sendInvalidationEvent(String name, Data key, String sourceUuid);
+
+    boolean wanReplicationEnabled(String cacheName);
```

Comment:
We may also get `wanReplicationEnabled` status from `ICacheRecordStore` (`ICacheRecordStore` gets it through `ICacheService`) also. So since we already access `ICacheRecordStore` with this way, we can find the `wanReplicationEnabled` status without another lookup inside `ICacheService`.